overlord/devicestate/devicestate_test.go

```diff
@@ -248,6 +248,11 @@ func (s *deviceMgrBaseSuite) settle(c *C) {
 	c.Assert(err, IsNil)
 }
 
+func (s *deviceMgrBaseSuite) longSettle(c *C) {
+	err := s.o.Settle(settleTimeout * 2)
```

Comment:
No strong opinion but why not make the settle timeout just longer for everything?

Reply:
+1

Reply:
This is done now.